osquery/logger/plugins/aws_firehose.cpp

```diff
@@ -107,15 +107,14 @@ Status FirehoseLogForwarder::setUp() {
   // Make sure we can connect to designated stream
   Aws::Firehose::Model::DescribeDeliveryStreamRequest r;
   r.SetDeliveryStreamName(FLAGS_aws_firehose_stream);
-  auto result = client_->DescribeDeliveryStream(r).GetResult();
-  auto description = result.GetDeliveryStreamDescription();
-  auto stream_name = description.GetDeliveryStreamName();
-  if (stream_name.empty()){
-      return Status(1,
-              "Could not find Firehose stream: " + FLAGS_aws_firehose_stream);
+  
+  auto outcome = client_->DescribeDeliveryStream(r);
+  if (outcome.IsSuccess()){
```

Comment:
this is a super nitpicky thing, but most often, in the osquery codebase we'll do:

```cpp
auto status = operation();
if (!status) {
  // bad stuff happened
  return 1;
}
// everything went as planned
return 0;
```

Do you mind just flipping this logic so to be:

```cpp
auto outcome = client_->DescribeDeliveryStream(r);
if (!outcome.IsSuccess()) {
  return Status(1, "...");
}
return Status(0, "...");
```